lib/augmentConfig.js

```diff
@@ -1,3 +1,4 @@
+/* eslint no-unused-vars: ["error", { "ignoreRestSiblings": true }] */
```

Comment:
Don't use this on top of file, it is bad practice, use `// eslint-disable-next-line` if you need ignore something on line or use `eslint-disable`/`eslint-enable` only between lines where it is unnecessary

Reply:
OK, I think `// eslint-disable-next-line no-unused-vars` is enough in this case. I'll revert and squash it.

Reply:
Done.

Reply:
@ybiquitous I think we can add it to shared ESLint config. Because rule does make sense. For now we can also have it in this package ESLint configuration until new config is ready.

Reply:
@hudochenkov Thanks for your suggestion, but I have no strong opinion on this case. 😅 

If there are some agreements, can I open a new issue on [eslint-config-stylelint](https://github.com/stylelint/eslint-config-stylelint)?

Reply:
I opened a new issue: https://github.com/stylelint/eslint-config-stylelint/issues/70